README.md

```diff
@@ -57,6 +57,18 @@ docker run -t -i -p 8080:8080 cockroachdb/cockroach shell
 # root@82cb657cdc42:/cockroach#
 ```
 
+If the docker command fails but Docker is installed, you probably need to initialize it.
```

Comment:
It would be good to have the exact error message here to help people who are searching for it. Do you remember what it was?